### DISCUSSION TITLE
How to use CLI commands in Docker image?

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#2208] [Q&A]
Hello, a friend of mine has installed Ente as a Docker image.
He is trying to Update the subscription for a user as described in here > [https://github.com/ente-io/ente/blob/main/cli/docs/generated/ente_admin_update-subscription.md](url)

How can this command be executed in the docker image installation?

**Comment by mnvr** [comment]
CLI itself is not contained in the Docker image. The Docker image contain's Ente's server.

Somewhat confusingly, there is _also_ support for running the CLI itself in a (separate) Docker image, but that is unrelated to the server's Docker image. I think the simplest option might be to just download the binary releases of the CLI and run them directly on your machine - https://github.com/ente-io/ente/tree/main/cli#install (You can run the CLI anywhere, not necessarily on the host that is running the server)
